package.json

```diff
@@ -79,6 +79,7 @@
     "@arbitrum/nitro-contracts": "1.1.1",
     "@arbitrum/token-bridge-contracts": "1.0.0-beta.0",
     "@gnosis.pm/safe-contracts": "1.3.0",
+    "@offchainlabs/upgrade-executor": "1.1.1",
```

Comment:
thinking if having `governanceDeployer.ts` to deploy UE using released bytecode would be an overkill

Reply:
if we do that we should also change the actions take the executor impl address as a constructor argument right

Reply:
not sure what you mean, it seems to already taking an address of the logic contract

Reply:
it just takes proxy admins at the moment and deploys the implementation in the action's constructor